finals-2024-01-16/code/5048-dijana-damcevska/src/app/models/mapping.ts

```diff
@@ -0,0 +1,57 @@
+import { Band, Album, Place } from "./band";
```

Comment:
The point of the mapping is to transform the data to a better format. In this case both formats are the same, so the transformation is in place. (I.e. the frontend could have split the location into city and country in the mapping) Still, the code is ok, +5 points